src/components/EmojiPicker/EmojiPickerMenu/getItemType.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Improves FlashList's recycling when there are different types of items
+ * @param {Object} item
+ * @returns {String}
+ */
+function getItemType(item) {
+    // item is undefined only when list is empty
+    if (!item) {
+        return;
+    }
+    if (item.header) {
+        return 'header';
```

Comment:
According to the FlashList docs, this method is called very often so it should be as fast as possible. I'd stick to if statement here instead of looping through an array.

Created a constant for available types ✅